apps/ringtones/js/manage_template.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+/* global Tagged */
+(function(exports) {
+  exports.templates = {
```

Comment:
Right now the same page will access the templates blindly. I'll update this to ```var Templates``` for now, and we can revisit it if needed.